lib/api/core/janitor.js

```diff
@@ -52,44 +52,38 @@ class Janitor {
   /**
    * Load roles, profiles and users fixtures into Kuzzle
    *
-   * @param {String} filePath
+   * @param {object} securities
    * @returns {Promise}
    */
-  loadSecurities (filePath) {
-    return this._loadJson(filePath)
-      .then(securities => {
-        return this._createSecurity('createOrReplaceRole', securities.roles)
-          .then(() => this._createSecurity('createOrReplaceProfile', securities.profiles))
-          .then(() => this._deleteUsers(securities.users))
-          .then(() => this._createSecurity('createUser', securities.users));
-      });
+  loadSecurities (securities) {
+    return this._createSecurity('createOrReplaceRole', securities.roles)
+      .then(() => this._createSecurity('createOrReplaceProfile', securities.profiles))
+      .then(() => this._deleteUsers(securities.users))
```

Comment:
If I understand this code well, this means that if a security fixture is used, all existing users are deleted?
Is this the wanted behavior?

Reply:
It's only deletes the users listed in the security fixture.  
So if you have users in your securities fixtures and you update them, you can reload the updated fixtures

Reply:
Oh ok :+1: